src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -732,8 +732,13 @@ def __call__(
         return_dict: bool = True,
         deterministic: bool = True,
         init_cache: bool = False,
+        input_embeds=None
```

Comment:
Can we keep this arg name consistent with the PyTorch version?
https://github.com/huggingface/transformers/blob/c60dd98e87373e7f0f5af29f3d49411c2e81fb69/src/transformers/models/t5/modeling_t5.py#L898

```suggestion
        inputs_embeds=None
```